flow/data_pipeline/query.py

```diff
@@ -17,6 +20,22 @@
         ]
     },
     "fact_energy_trace": {},
+    "fact_safety_metrics": {
+        "fact_safety_metrics_agg": [
+            "FACT_SAFETY_METRICS_AGG"
+        ]
+    },
+    # @brent: are these needed? Is there a race condition here that may break things?
+    # "fact_safety_metrics_agg": {
+    #     "leaderboard_chart": [
+    #         "LEADERBOARD_CHART"
+    #     ]
+    # },
+    # "fact_network_throughput_agg": {
+    #     "leaderboard_chart": [
+    #         "LEADERBOARD_CHART"
+    #     ]
+    # }
```

Comment:
we don't need this for now because `fact_network_fuel_efficiency_agg` should in-theory be run after both  `fact_safety_metrics_agg` and `fact_network_through_agg`. But this is not a hard guarantee since S3 events are indeed asynchronous. I'll be implementing a better way of doing this but in the meantime, we can use the current version first.